tests/etl/test_transformers.py

```
@@ -0,0 +1,78 @@
+
+class DummySchema:
+    def __init__(self):
+        self.columns = {
+            "a": DummyColumn("int64"),
+            "b": DummyColumn("float64"),
+            "c": DummyColumn("datetime64[ns]"),
+            "d": DummyColumn("object"),
+        }
+
+
+def build_transformer() -> transformers.KaggleTransformer:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)